sdk/sdk-api/src/main/java/mil/emp3/api/abstracts/Feature.java

```diff
@@ -429,4 +433,20 @@ protected double makePositive(double dValue, String message) {
             return Math.abs(dValue);
         }
     }
+
+    /**
+     * Convert selected members to String.
+     * @return
+     */
+    @Override
+    public String toString() {
+        String str = "Feature " + getFeatureType() + "\n";
+        str += "PathType " + getPathType().toString() + " Extrude " + getExtrude() + " AltitudeMode " + getAltitudeMode() + "\n";
+        str += EmpStyles.toString(getFillStyle()) + "\n" + EmpStyles.toString(getStrokeStyle()) + "\n";
+        for(IGeoPosition p: getPositions()) {
+            str += EmpGeoPosition.toString(p) + "\n";
+        }
+        str += "Feature Done\n";
+        return str;
+    }
```

Comment:
The "Feature" at the beginning and the "Feature Done" at the end are a given. Start the string with the feature type followed by the Name. As for the position list I would not include it. It can get very long and they are double values.